sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ExpressionEvalHelper.scala

```diff
@@ -79,7 +79,6 @@ trait ExpressionEvalHelper {
         fail(
           s"""
             |Code generation of $expression failed:
-            |${evaluated.code}
```

Comment:
@davies I moved the code snippet to `CodeGenerator`, as `evaluated.code` just show part of the code, sometimes we do need the `declareMutableStates` code for troubleshooting.